snapcraft/storeapi/metrics.py

```diff
@@ -0,0 +1,166 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2021 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import enum
+from typing import Any, Dict, List, Optional, Union
+
+import attr
+
+"""
+This module holds representations for request and results for the metrics API
+endpoint provided by the Snap Store.
+
+The relevant API documentation is available at:
+https://dashboard.snapcraft.io/docs/api/snap.html?#fetch-metrics-for-snaps
+"""
+
+
+MetricsStatus = enum.Enum(  # type: ignore
+    value="MetricsStatus",
+    names=[("OK", "OK"), ("FAIL", "FAIL"), ("NO DATA", "NO DATA")],
+)
```

Comment:
Can't this be declared as a class, for readability and consistency?

Reply:
Unfortunately we cannot AFAIK because "NO DATA" has a space. :-(